src/python/dxpy/scripts/dx.py

```diff
@@ -4669,38 +4667,37 @@ def register_parser(parser, subparsers_action=None, categories=('other', ), add_
 #####################################
 parser_compile_dxni = subparsers.add_parser(
     'compile_dxni',
-    help='Dx Native call Interface',
-    description= fill('Dx Native call Interface. Create stubs for calling dx '
+    help='Dx Native Call Interface',
+    description= fill('Dx native call interface (DxNI). Create stubs for calling dx '
                       'executables (apps/applets/workflows), and store them as WDL '
                       'tasks in a local file. Allows calling existing platform executables '
                       'without modification. Default is to look for applets.'),
     prog= 'dx compile_dxni')
 
 # positional argument: output file
-parser_compile_dxni.add_argument("outputFile",
+parser_compile_dxni.add_argument("output_file",
```

Comment:
What I meant was `add_argument("output-file", dest="output_file", ...` a bit like [here](https://github.com/dnanexus/dx-toolkit/blob/b8742dd77ffd5ef9852cfcfecd4cf13fb60b90b6/src/python/dxpy/scripts/dx.py#L4373). Sorry for nitpicking but this will stick, thanks :)

Reply:
Right, I tried that, however, it is supported only for optional arguments. It does not work for  positional arguments, those that do not start with a dash.